presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -102,8 +102,66 @@ Property Name                                      Description
 ``hive.immutable-partitions``                      Can new data be inserted into existing partitions?           ``false``
 
 ``hive.max-partitions-per-writers``                Maximum number of partitions per writer.                     100
+================================================== ============================================================ ==========
+
+Amazon S3 Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The Hive connector also allows querying data stored in Amazon S3. The SQL support 
+for S3 tables is the same as for HDFS tables.
```

Comment:
I'm realizing based on your comment on the testing pull request that the sql support is not the same because you can't do CREATE TABLE or CTAS.